csharp-basic-topics/TypesOfInheritanceInCSharp/Tests/TypesOfInheritanceTest.cs

```diff
@@ -0,0 +1,119 @@
+using System;
+using System.IO;
+using TypesOfInheritanceInCSharp;
+using Xunit;
+
+namespace TypesOfInheritanceInCSharpTests;
+public class TypesOfInheritanceTest
+{
+    [Fact]
+    public void WhenHaveSmartphone_ThenItIsMobileDevice()
+    {
+        var smartphone = new Smartphone();
+
+        Assert.True(smartphone is MobileDevice);
+    }
+
+    [Fact]
+    public void WhenHaveAndroidsmartphone_ThenItIsSmartphone()
+    {
+        var smartphone = new AndroidSmartphone();
+
+        Assert.True(smartphone is Smartphone);
+    }
+
+    [Fact]
+    public void WhenHaveAndroidsmartphone_ThenItIsMobileDevice()
+    {
+        var smartphone = new AndroidSmartphone();
+
+        Assert.True(smartphone is MobileDevice);
+    }
+
+    [Fact]
+    public void WhenHaveConvertibleNotebook_ThenItIsMobileDevice()
+    {
+        var smartphone = new ConvertibleNotebook();
+
+        Assert.True(smartphone is MobileDevice);
+    }
+
+    [Fact]
+    public void WhenHaveConvertibleNotebook_ThenItIsLaptop()
+    {
+        var smartphone = new ConvertibleNotebook();
+
+        Assert.True(smartphone is Laptop);
+    }
+
+    [Fact]
+    public void WhenHaveSmartphone_ThenItOverridesDeviceCanMakePhoneCall()
+    {
+        var smartphone = new Smartphone();
+        smartphone.OperatingSystem = "iOS";
+        smartphone.Inches = 10;
+
+        Assert.True(smartphone.DeviceCanMakePhoneCall());
```

Comment:
Add newlines between Arrange/Act/Assert wherever applicable.

Reply:
I don't know if this means that I need to write var result = smartphone.DeviceCanMakePhoneCall(); and then Assert.True(result)